pkg/skaffold/deploy/deploy.go

```diff
@@ -21,9 +21,18 @@ import (
 	"fmt"
 	"io"
 
+	"k8s.io/apimachinery/pkg/runtime"
+
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
 )
 
+// DeployResult is a holder struct encapsulating the deployed k8s objects
+type DeployResult struct {
+	obj       *runtime.Object
+	namespace string
```

Comment:
The namespace is part of of the runtime.Object I believe

Reply:
unfortunately it isn't always, helm can't be trusted to put it where it should be, among many other things that it should do but doesn't